kyuubi-server/src/main/scala/org/apache/kyuubi/engine/ProcBuilder.scala

```diff
@@ -148,6 +150,38 @@ trait ProcBuilder {
     proc
   }
 
+  def execKillYarnJob(appId: String): Int = {
+    val hadoopHomeOpt = env.get("HADOOP_HOME").orElse {
+      throw KyuubiSQLException(s"Failed to kill yarn job. HADOOP_HOME is not set! " +
```

Comment:
Should we also find `/usr/bin/yarn`?

Reply:
I think get the absolute path is enough, how do you concider it ?

Reply:
Yeah, I'm fun with that.